internal/integrations/github/github.go

```diff
@@ -0,0 +1,53 @@
+package github
+
+import (
+	"context"
+	"errors"
+
+	"github.com/google/go-github/v26/github"
+	"golang.org/x/oauth2"
+)
+
+var errTravisCIBuildNotFound = errors.New("could not find a TravisCI build for specified ref")
+
+// ChecksService wraps github.ChecksService
+type ChecksService interface {
+	ListCheckRunsForRef(context.Context, string, string, string, *github.ListCheckRunsOptions) (*github.ListCheckRunsResults, *github.Response, error)
+}
+
+// Github handles integrations with the Github API
+type Github struct {
+	Org    string
+	Client *github.Client
```

Comment:
The `Client` field is unused, should it be removed?

Reply:
`Client` field init was removed from `New`, but the field is still declared here.